custom_components/hacs/base.py

```diff
@@ -1076,6 +1081,13 @@ async def async_update_downloaded_custom_repositories(self, _=None) -> None:
             ):
                 self.queue.add(repository.update_repository(ignore_issues=True))
 
+        async def update_coordinators() -> None:
+            """Update all coordinators."""
+            for coordinator in self.coordinators.values():
+                coordinator.async_update_listeners()
+
+        self.queue.add(update_coordinators())
```

Comment:
This will execute too soon without https://github.com/hacs/integration/pull/3571

Reply:
Looks a bit precarious to add a coroutine instead of adding a coroutine function. Why is the interface like that?

Reply:
The queue is a queue of tasks (type restricted to coroutines), I *think* that is a sane interface?

Reply:
OK, I see the rationale in https://github.com/hacs/integration/pull/3571#discussion_r1555639357, I guess that makes sense.